theme-looper-tests.el

```diff
@@ -130,9 +130,9 @@
           ;;Should apply customizations when specified
           (load-theme 'tango
                       t)
-	      (theme-looper-set-post-switch-script (lambda ()
-                                                 (set-face-background 'default
-                                                                      "green")))
+	      (add-hook 'theme-looper-post-switch-hook
+                    (lambda ()
+                      (set-face-background 'default "green")))
```

Comment:
Using hooks is a great idea. It never occurred to me, with my shallow experience with packages.